src/agent/LibFuzzerDotnetLoader/Program.cs

```diff
@@ -81,4 +75,73 @@ static bool TryTestOne<T>(MethodInfo method, Func<T, SharpFuzz.ReadOnlySpanActio
             return TryTestOne<TestOneArray>(method, t => span => t(span.ToArray()));
         }
     }
+
+    class LibFuzzerDotnetTarget
+    {
+        public string AssemblyPath { get; }
+        public string ClassName { get; }
+        public string MethodName { get; }
+
+        public LibFuzzerDotnetTarget(string assemblyPath, string className, string methodName)
+        {
+            AssemblyPath = assemblyPath;
+            ClassName = className;
+            MethodName = methodName;
+        }
+
+        public static LibFuzzerDotnetTarget FromEnvironment()
+        {
+            try {
+                return FromEnvironmentVarDelimited();
+            }
+            catch
+            {}
+
+            try {
+                return FromEnvironmentVars();
+            }
+            catch
+            {}
+
+            throw new Exception("No fuzzing target specified by environment variables");
+        }
+
+        static LibFuzzerDotnetTarget FromEnvironmentVars()
+        {
+            var assemblyPath = Environment.GetEnvironmentVariable(EnvVar.ASSEMBLY);
+            var className = Environment.GetEnvironmentVariable(EnvVar.CLASS);
+            var methodName = Environment.GetEnvironmentVariable(EnvVar.METHOD);
+
+            var missing = new List<string>();
+
+            if (assemblyPath is null) { missing.Add(EnvVar.ASSEMBLY); }
+            if (className is null) { missing.Add(EnvVar.CLASS); }
+            if (methodName is null) { missing.Add(EnvVar.METHOD); }
+
+            if (assemblyPath is null || className is null || methodName is null)
```

Comment:
I'm doing this so the compiler can understand that these variables are non-`null` on non-exceptional procedure exit. As you'd imagine, it can't infer that from a `missing.Count` check.